src/styles/blocks/watch.scss

```diff
@@ -0,0 +1,53 @@
+.watch {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+
+  background-color: $darkColor;
+
+  &__dial {
+    @include circle(400px);
+
+    position: relative;
+    border: 20px solid $colorOfWatch;
+
+    background-color: rgba(23, 22, 22, 0.82);
+  }
+
+  &__center {
+    @include circle(25px);
+
+    position: absolute;
+    left: 50%;
+    top: 50%;
+    transform: translate(-50%,-50%);
+    z-index: 4;
+
+    background-color: $colorOfWatch;
+  }
+
+  &__arrow-hour {
+    @include arrowStyles(110px, 8px);
+    animation-duration: 200s;
+  }
+  &__arrow-minute {
+    @include arrowStyles(160px, 8px);
+    animation-duration: 60s;
```

Comment:
I would recommend you to count time properly.
For example minute hand  supposed to have animation-duration 3600s
and second hand 60sec to show time correctly